src/pj_fwd.c

```diff
@@ -54,18 +54,46 @@ PJ_COORD pj_fwd_prepare (PJ *P, PJ_COORD coo) {
         if (lp.phi < -M_HALFPI)
             lp.phi = -M_HALFPI;
 
+
+        coo.lp = lp;
```

Comment:
May I suggest calling it `coord` instead. I think it reads better.

Reply:
`coo` has been conventionally used here, there, and everywhere:

    grep coo *.c *.h | grep -v coord | wc -l

gives 74 hits.

`coord` is used in methods and type names (`PJ_COORD`, `proj_coord ()`, etc.) - `coo` is used for an otherwise unspecific instance, intentionally modelled after the equivalent unspecificness of the versatile variable `foo` of `foo, bar, baz` fame.

`coord` is used as an instance name a few places in `proj_4D_api.c` and `gie.c` - I would be less reluctant renaming these instances to `coo`, than changing the already widespread `coo` idiom.

Reply:
> coo has been conventionally used here, there, and everywhere

I know, but I don't like the convention. It somehow made sense previously when `coo` was also used in the now gone `PJ_OBS` where a short name was justified. Since that no longer exists I think it is perfectly reasonable to use a more descriptive convention.

I would also object to using non-descriptive names like `foo` and `bar` in anything other than example code so I am not buying that argument.

Anyway, my point is just that `coord` makes more sense to readers unfamiliar with the code-base as a whole. I am not suggesting that the existing occurences of `coo` be replaced now, I just wanted to raise a flag here since it has been bothering me for a while. Do what you feel is best - ultimately it is in the nitpicking department!

Reply:
@kbevers: Since you dislike the convention, I think we should consider reconsidering the convention(s).

From classic PROJ/Gerald, we have **`P`** for the generic `PJ` object. From 3D-PROJ/Frank, we have **`ctx`** for the generic `PJ_CONTEXT` object. So apparently, the smaller the object, the longer the name.

With that in mind, it actually makes quite a lot of sense to switch **`coo`** (same length as `ctx`, but smaller) to your preferred **`coord`** (longer and smaller than `ctx`).

But if that's to be done I suggest we do it consequently, and reserve a rainy summer day, in time for 5.1.0, for a sweep-over of the entire code base, switching `coo` to `coord` - and collecting a list of other *paper cuts*/minor annoyances to be remedied at the same occasion.

I'll buy the beer, and open the issue for discussion - you bring the stamina, and I'll try to follow your typing pace :-)